### PR TITLE
Add test for lttng tracing

### PR DESCRIPTION
**Pull request by tmds**
Fixes https://github.com/redhat-developer/dotnet-regular-tests/issues/48

CC @RheaAyase @omajid

### CODE REVIEW

**Comment by tmds** [comment]
@RheaAyase can I please get some more permissions on this repo, like being able to request a review?

File: lttng/test.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+echo "== Starting lttng session"
+start_session
+
+# Create new console application to generate an event
+echo "== Creating new console application"
+export COMPlus_PerfMapEnabled=1
+export COMPlus_EnableEventLog=1
+dotnet new console -o $TEST_FOLDER/console &
+DOTNET_PID=$!
+wait $DOTNET_PID
```
Comment (omajid) [inline lttng/test.sh]: Can these 3 lines be collapsed into just `dotnet new console -o $TEST_FOLDER/console`?

**Comment by omajid** [comment]
@tmds Have you tests this on RHEL 7 or 8? As far as I can tell, unsupported versions of `babeltrace` exist in some RHEL 8 repos, but not `lttng` or `lttng-sessiond`. I guess we will need some third-party RPMs for that?

File: lttng/test.sh (hunk 1 above)
Comment (tmds) [reply lttng/test.sh]: I wrote it as separate lines to get the pid of the dotnet process. Not sure how to get it without putting the dotnet process in background first.

**Comment by tmds** [comment]
I wrote and ran the test on Fedora. I installed lttng-tools and babeltrace packages.

RHEL 7 packages are available at: http://packages.efficios.com/

File: lttng/test.sh (hunk 1 above)
Comment (omajid) [reply lttng/test.sh]: Oh. Right. Sorry about the noise.

**Comment by tmds** [comment]
@omajid thanks for reviewing.
@RheaAyase please merge this if it is good for you too.

**Pull request merged**